h2o-logger/src/main/java/water/logging/LoggerFactory.java

```diff
@@ -15,7 +20,21 @@
      * @return Slf4JLogger if SLF4J is on the classpath, ConsoleLogger if not
      */
     public static Logger getLogger(Class<?> clazz) {
-        if (isSlf4JAvailable()) {
+        if (isSlf4JAvailable(DEFAULT_CLASS_TO_CHECK)) {
+            return new Slf4JLogger(clazz);
+        } else {
+            return new ConsoleLogger();
+        }
+    }
+
+    /**
+     * Returns new logger for each invocation depending on whether custom class is available.
+     *
+     * @param clazz class from which getLogger() is called
+     * @return Slf4JLogger if SLF4J is on the classpath, ConsoleLogger if not
+     */
+    Logger getCustomLogger(Class<?> clazz) {
```

Comment:
this method is tested but not used anywhere in the code, therefore should be removed